doc/en/assert.rst

```diff
@@ -174,7 +175,9 @@ if you run this module:
     $ pytest test_assert2.py
     =========================== test session starts ============================
     platform linux -- Python 3.x.y, pytest-4.x.y, py-1.x.y, pluggy-0.x.y
+    hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('$REGENDOC_TMPDIR/.hypothesis/examples')
```

Comment:
hmm, maybe we should _uninstall_ hypothesis in regendoc :thinking:

Reply:
Indeed, but PYTEST_ADDOPTS... oh. Weird, how has this been working until now??

Reply:
heh, I suspect most have released with old regendoc envs sitting around?

iirc when I ran the releases I ran I manually uninstalled hypothesis to try and minimize the diff but then forgot to make an issue about it :man_shrugging:

Reply:
Can you please create one then? I'm uninstalling hypothesis manually as well and regenerating the docs.

Reply:
done! https://github.com/pytest-dev/pytest/issues/4602

Reply:
Thanks! Pushed the new regendocs without hypothesis output. 👍